core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -8553,6 +8553,13 @@ public void subTestIntervalSecondFailsValidation() {
         .ok("('100' IS NOT JSON SCALAR)");
   }
 
+  @Test public void testExtractValue() {
```

Comment:
remove this test. it is not necessary. you have not changed the parser.

Reply:
Done :)